ml_metadata/metadata_store/BUILD

```diff
@@ -647,3 +647,40 @@ cc_test(
         "@org_tensorflow//tensorflow/core:test",
     ],
 )
+
+ml_metadata_go_wrap_cc(
```

Comment:
qq, will you only depend on the proto targets? if so, shall we exclude these targets?

Some background, the go libs were excluded temporarily due to the go-rules had issues with py3.8 / py3.9 integration testing images. The grpc, proto dependencies currently were introduced by TF workspace. We had a todo to reintroduce them back once drop the TF's dependency in the WORKSPACE.